tests/web/classifier_service/dummy_classifier.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+from typing import Any, Dict, Hashable, Iterator, Iterable, Mapping, Sequence, Union
+
+from smqtk_classifier import (
+    ClassifyDescriptorSupervised,
+    ClassifyDescriptor
+)
+from smqtk_descriptors import DescriptorElement
+
+
+STUB_CLASSIFIER_MOD_PATH = __name__
+
+
+class DummyClassifier (ClassifyDescriptor):
+    def get_config(self) -> Dict[str, Any]:
+        """
+        Return a JSON-compliant dictionary that could be passed to this
+        class's ``from_config`` method to produce an instance with identical
+        configuration.
+
+        In the common case, this involves naming the keys of the dictionary
+        based on the initialization argument names as if it were to be passed
+        to the constructor via dictionary expansion.
+
+        :return: JSON type compliant configuration dictionary.
+        :rtype: dict
+
+        """
+        # No constructor, no config
+        return {}
```

Comment:
Since this testing class method is never used, it should follow a similar pattern to `DummySupervisedClassifier` and define this like
```python
def get_config(self) -> Dict[str, Any]: ...
```